sdk/ai/azure-ai-inference/README.md

```diff
@@ -0,0 +1,231 @@
+# Azure Model client library for Java
+
+Azure Model client library for Java.
+
+This package contains Microsoft Azure Model client library.
+
+## Documentation
+
+Various documentation is available to help you get started
+
+- [API reference documentation][docs]
+- [Product documentation][product_documentation]
+
+## Getting started
+
+### Prerequisites
+
+- [Java Development Kit (JDK)][jdk] with version 8 or above
+- [Azure Subscription][azure_subscription]
+
+### Adding the package to your product
+
+[//]: # ({x-version-update-start;com.azure:azure-ai-inference;current})
+```xml
+<dependency>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-ai-inference</artifactId>
+    <version>1.0.0-beta.1</version>
+</dependency>
+```
+[//]: # ({x-version-update-end})
+
+### Authentication
+
+In order to interact with the Azure AI Inference Service you'll need to create an instance of client class,
+[ChatCompletionsAsyncClient][chat_completions_client_async] or [ChatCompletionsClient][chat_completions_client_sync] by using
+[ChatCompletionsClientBuilder][chat_completions_client_builder]. To configure a client for use with
+Azure Inference, provide a valid endpoint URI to an Azure Model resource along with a corresponding key credential,
+token credential, or [Azure Identity][azure_identity] credential that's authorized to use the Azure Model resource.
+
+#### Create a Chat Completions client with key credential
+Get Azure Model `key` credential from the Azure Portal.
+
+```java readme-sample-createSyncClientKeyCredential
+ChatCompletionsClient client = new ChatCompletionsClientBuilder()
+    .credential(new AzureKeyCredential("{key}"))
+    .endpoint("{endpoint}")
+    .buildClient();
+```
+or
+```java readme-sample-createAsyncClientKeyCredential
+ChatCompletionsAsyncClient client = new ChatCompletionsClientBuilder()
+    .credential(new AzureKeyCredential("{key}"))
+    .endpoint("{endpoint}")
+    .buildAsyncClient();
+```
+
+#### Create a client with Azure Active Directory credential
+Azure SDK for Java supports an Azure Identity package, making it easy to get credentials from Microsoft identity
+platform.
+
+Authentication with AAD requires some initial setup:
+* Add the Azure Identity package
+
+[//]: # ({x-version-update-start;com.azure:azure-identity;dependency})
+```xml
+<dependency>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-identity</artifactId>
+    <version>1.13.1</version>
+</dependency>
+```
+[//]: # ({x-version-update-end})
+
+After setup, you can choose which type of [credential][azure_identity_credential_type] from azure.identity to use.
+As an example, [DefaultAzureCredential][wiki_identity] can be used to authenticate the client:
+Set the values of the client ID, tenant ID, and client secret of the AAD application as environment variables:
```

Comment:
Is service principal the recommended auth type @billwert , @g2vinay ?

Reply:
We should recommend managed identity if anything. For this documentation it is fine to just recommend `DefaultAzureCredential` without further suggestion. I'd just remove this entire paragraph.

I see you've probably just copied this from another readme as I can see this string in a handful of other places. Opened #41890 to track this.

Reply:
Removed